src/components/pages/weather/WeatherPage.css

```diff
@@ -0,0 +1,8 @@
+.column {
+  display: inline-block;
+}
+
+.cell {
+  border: 1px solid black;
+  min-height: 25px;
+}
```

Comment:
I would probably err on the side of caution and call these `weather-column` and `weather-cell` just to avoid potential name clashes elsewhere on the site.

Even if you're unlikely to reuse them it's generally good practice to be as specific as possible in your css selector names :)